oracles/oracles.md

```diff
@@ -16,6 +16,8 @@
 
 ## [Oracle life cycle examples](./oracle_life_cycle.md)
 
+## [Oracle WebSocket API](./oracle_ws_api.md)
```

Comment:
When we spoke today I understood we wanted to keep epoch API in this repo though segregated somehow. I would propose last section in this document `## epoch node API` and separate folder `./epoch_api` containing API docs.